xt/lib/Pherkin/Extension/pageobject_steps/nav_steps.pl

```diff
@@ -90,6 +91,11 @@
     'overhead entry' => 'PageObject::App::Parts::Overhead',
     'system defaults' => 'PageObject::App::System::Defaults',
     'system taxes' => 'PageObject::App::System::Taxes',
+    'year-end' => ' PageObject::App::Closing',
+    'Enter Inventory' => ' PageObject::App::Parts::AdjustSetup',
+    'GS Search' => ' PageObject::App::Search::GoodsServices',
+    'RPT Balance Sheet' => ' PageObject::App::Report::Filters::BalanceSheet',
+    'TA Inventory' => ' PageObject::App::Parts::AdjustSearchUnapproved',
```

Comment:
I'm not really happy with `TA Inventory` as it would result in `Then I expect to see the TA Inventory screen` which doesn't ring a bell for me and probably many other users. Maybe `Then I expect to see the unapproved inventory adjustments search screen` would make more sense?